pysc2/env/sc2_env.py

```diff
@@ -176,7 +176,7 @@ def _setup(self,
     screen_size_px = point.Point(*screen_size_px)
     minimap_size_px = point.Point(*minimap_size_px)
     interface = sc_pb.InterfaceOptions(
-        raw=visualize, score=True,
```

Comment:
Add a constructor flag for whether to expose the raw features, and condition this on that.

Reply:
Done